### PR TITLE
Update gops to v0.3.18 and build it statically linked

### PR DESCRIPTION
**Pull request by tklauser**
See commit messages for details.

### CODE REVIEW

File: images/cilium-test/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Copyright 2020-2021 Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-ARG CILIUM_BUILDER_IMAGE=quay.io/cilium/cilium-builder:3203db544891bf8360d97085ecce89282f04c78d@sha256:c648e3e2f9f99b7e53922691cfea7515ebeb7aad273f9738f00103494dabcc32
+ARG CILIUM_BUILDER_IMAGE=quay.io/cilium/cilium-builder:5883692e7dfb1d7b96f31c710d4ef31e6e812654
```
Comment (pchaigno) [inline images/cilium-test/Dockerfile]: Why is the SHA256 hash dropped here and in `test/k8sT/manifests/test-verifier.yaml`?

Comment (tklauser) [reply images/cilium-test/Dockerfile]: I'm still in the process of building updated images (see https://docs.cilium.io/en/latest/contributing/development/images/#update-cilium-builder-and-cilium-runtime-images). They will be added again in the final iteration.

Comment (tklauser) [reply images/cilium-test/Dockerfile]: All images built now, the hashes should be updated as well.

**Review by pchaigno — APPROVED** [review]

**Comment by tklauser** [comment]
test-me-please

**Comment by tklauser** [comment]
test-gke

**Review by aanm — APPROVED** [review]

**Comment by tklauser** [comment]
test-me-please

**Comment by tklauser** [comment]
test-gke

**Comment by tklauser** [comment]
test-gke

**Comment by tklauser** [comment]
test-runtime

**Pull request merged**